Wp 104 add client side tests for events api

aldafu opened the pull request:
http://jira.webinos.org/browse/WP-104

aldafu committed:
Add client side tests for Events API.

This test spec is written using jasmine.

Jira-issue: WP-104

aldafu committed:
Add jasmine spec runner.

Add the necessary files to run jasmine spec tests. E.g. needed by the
tests for the Events API.

Jira-issue: WP-104

habibvirji commented on webinos/test/spec/EventsSpec.js:
```diff
@@ -0,0 +1,112 @@
+describe("Events API", function() {
```
missing webinos header

aldafu replied:
added

aldafu committed:
Add event sending, receiving tests.

The tests now cover creating events, onError callback if sending fails,
succesful sending and delivering of events and receiving events.
Added license header.

Jira-issue: WP-104
